apps/safe-claiming-app/src/widgets/ClaimingWidget.tsx

```diff
@@ -140,22 +140,29 @@ const ClaimingWidget = () => {
     <>
       <div>
         <Title>Your voting power</Title>
-        <Box
-          display="flex"
-          alignItems="center"
-          justifyContent="center"
-          gap={1}
-          mb={1}
+        <Link
```

Comment:
To me it's a bit surprising that the Voting power is now clickable and leads to the claiming app.

At least without any visual indicator that this is clickable. Probably we should use `ButtonBase` or something similar as a wrapper instead of link to clarify that this is clickable?